File: Producto/history-service/src/main/java/com/tagok/history_service/domain/PorticoRuta.java
package com.tagok.history_service.domain;

import lombok.Data;

@Data
public class PorticoRuta {
    private String nombre;
    private String codigo;
    private String autopista;
    private String codigoAutopista;
    private double longitud;
    private double latitud;
    private String tipoTarifa;
    private double valor;
    private String fechaHora;
}

File: Producto/tag-ok-app/app/src/main/java/com/tagok/app/data/dto/PorticoRuta.kt
package com.tagok.app.data.dto

import kotlinx.serialization.Serializable

@Serializable
data class PorticoRuta(
    val nombre: String? = "",
    val codigo: String = "",
    val sentido: String = "",
    val autopista: String = "",
    val codigoAutopista: String? = "",
    val longitud: Double = 0.0,
    val latitud: Double = 0.0,
    val tarifa: String = "",
    val valor: Double = 0.0)